native/src/components/EastereggImage.tsx

```diff
@@ -119,10 +119,15 @@ class EastereggImage extends React.Component<PropsType, StateType> {
   }
 
   render(): ReactNode {
+    const locationMarker = buildConfigAssets().locationMarker
+    if (!locationMarker) {
```

Comment:
I think just adding locationMarker with the logical && operator to the jsx would be shorter and nicer.